test/unit_test/passes/onnx/test_gptq.py

```diff
@@ -0,0 +1,75 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from pathlib import Path
+
+import pytest
+import torch
+
+from olive.hardware.accelerator import AcceleratorSpec, Device
+from olive.model.handler.pytorch import PyTorchModelHandler
+from olive.passes.olive_pass import create_pass_from_dict
+from olive.passes.onnx import GptqQuantizer
+
+
+@pytest.mark.skipif(
+    not torch.cuda.is_available(),
+    reason="gptq requires GPU.",
+)
+def test_gptq_default(tmp_path: Path):
```

Comment:
this is a nit: since the test code is mostly the same, maybe the two cases can be parameterized using a flag like `dataset_source` or something?